docs/docs/concepts/relation-tuples.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Relation Tuples
+---
+
+import RelationTuplePrism from '@theme/RelationTuplePrism'
+RelationTuplePrism()
+
+Relation tuples are the underlying datatype of Ory Keto's access control
+language. They encode relations between [objects](./objects) and
+[subjects](./subjects). A relation tuple is associated with a
+[namespace](./namespaces) where its relation has to be defined and configured.
+The following BCNF grammar describes the encoding used in the documentation and
```

Comment:
What is BCNF 😅 Maybe link to something?

Reply:
Note to self: this should refer to https://en.wikipedia.org/wiki/Backus%E2%80%93Naur_form